wpilib/wpilib/relay.py

```diff
@@ -71,6 +72,14 @@ class Value:
             "Reverse": kReverse,
         }
 
+        @classmethod
+        def getPrettyValue(cls, num):
```

Comment:
can we get docstrings for these two?

Reply:
I think these should just be changed to enums.

Reply:
Yeah... these should probably be `IntEnum`.

Reply:
ok. thought there was a reason we couldn't use them